persistence/src/main/java/org/mskcc/cmo/metadb/persistence/MetaDbSampleRepository.java

```diff
@@ -61,8 +61,16 @@
 
     @Query("MATCH (r: Request {requestId: $reqId}) "
             + "MATCH(r)-[:HAS_SAMPLE]->(sm: Sample) "
-            + "MATCH (sm)<-[:IS_ALIAS]-(s: SampleAlias {toLower(namespace): 'igoid', value: $igoId}) "
+            + "MATCH (sm)<-[:IS_ALIAS]-(s: SampleAlias {namespace: 'igoId', value: $igoId}) "
```

Comment:
this `toLower` neo4j method was throwing an exception so I've removed it. Since we're setting this field I think it's safe to just refer to the namespace in its proper casing